### PR TITLE
docs(readme): add status badges

### PR DESCRIPTION
**Pull request by kdsinghcoder**
Fixed issue: #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
...
+![GitHub pull requests](https://img.shields.io/github/issues-pr/Uniauth/uniauth-backend)
+![GitHub last commit](https://img.shields.io/github/last-commit/uniauth/uniauth-backend)
+![Gitlab code coverage](https://img.shields.io/gitlab/coverage/uniauth/uniauth-backend/master)
+![GitHub top language](https://img.shields.io/github/languages/top/uniauth/uniauth-backend)
+![Docker Automated build](https://img.shields.io/docker/automated/UniAuth/uniauth-backend)
+
 # Authentico
 
-[![Build Status](https://www.travis-ci.com/YashKumarVerma/authentico-backend.svg?token=bdYdpM7ki4qrmdCwJmGf&branch=master)](https://www.travis-ci.com/YashKumarVerma/authentico-backend) ![Ensure Build](https://github.com/YashKumarVerma/authentico-backend/workflows/Ensure%20Build/badge.svg)
+![Ensure Build](https://github.com/YashKumarVerma/authentico-backend/workflows/Ensure%20Build/badge.svg)
```
Comment (YashKumarVerma) [inline README.md:10]: please use the batch from uniauth/uniauth-backend and not my personal account

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]

**Review by kdsinghcoder — COMMENTED** [review]
Updated as per instructions

**Comment by YashKumarVerma** [comment]
please get all the batches in a single line, immediately after the title. 
Also, change the title to `UniAuth Backend`
image

**Comment by YashKumarVerma** [comment]
add
[CodeFactor](https://www.codefactor.io/repository/github/uniauth/uniauth-backend)

**Review by kdsinghcoder — COMMENTED** [review]
image

**Review by YashKumarVerma — APPROVED** [review]

**Comment by YashKumarVerma** [comment]
thanks !

**Pull request merged**